CMakeLists.txt

```diff
@@ -236,10 +262,10 @@ ENDIF()
 IF(NOT DISABLE_TESTS)
     ENABLE_TESTING()
     ADD_EXECUTABLE(hiredis-test test.c)
-    TARGET_LINK_LIBRARIES(hiredis-test hiredis)
+    TARGET_LINK_LIBRARIES(hiredis-test ${HIREDIS_DEFAULT_LIBRARY})
```

Comment:
I believe the same change is needed in `examples/CMakeLists.txt`, at least for windows.

Reply:
..and a  `HIREDIS_SSL_DEFAULT_LIBRARY`

Reply:
Indeed, I'll update the PR